plugins/github/api.ml

```diff
@@ -132,6 +170,11 @@ module Commit_id = struct
     let gref = Ref.to_git id in
     Current_git.Commit_id.v ~repo ~gref ~hash
 
+  let name t =
+    match String.split_on_char '/' t.owner_name with
+    | [owner;repo_name] -> Some (owner, repo_name)
+    | _ -> None
```

Comment:
Can this happen?

Reply:
It seems we always have both and this can't really fail in practice, the refactoring looks like this:
https://github.com/ocurrent/ocurrent/commit/5da9b0381ef9a1644a00db73d5f947745b8be938

Reply:
That looks good to me (with old code removed rather than commented).